src/shared/tests/file-glob.js

```diff
@@ -31,6 +33,7 @@ test('testGlob', t => {
   assertGlobFalse('a', 'ab');
   assertGlobFalse('a', 'ba');
   assertGlobFalse('a', 'bab');
+  // true because partial match (include the folder)
```

Comment:
I still need to go through and update this test file to indicate partial (possible parent directory) matches.